src/components/calculator/PointsDisplay.tsx

```diff
@@ -131,9 +131,8 @@ export default function PointsDisplay(props: {
     if (name === 'moderate') return t('calculator.category_moderate')
     if (name === 'high') return t('calculator.category_high')
     if (name === 'very-high') return t('calculator.category_very_high')
-    if (name === 'dangerously-high')
-      return t('calculator.category_dangerously_high')
-    if (name === 'extreme') return t('calculator.category_extreme')
+    if (name === 'dangerously-high' || name === 'extreme')
+      return t('calculator.category_extreme')
```

Comment:
I think you maybe have this backward? Dangerously high is below extreme. Here's how they each look: 

![image](https://user-images.githubusercontent.com/32290/128225647-a69a23f9-c532-4cca-a61e-4c55cf3293e8.png)
![image](https://user-images.githubusercontent.com/32290/128225674-f1c23830-512f-468e-8435-d3577769aa18.png)

Reply:
@beshaya so here's a screenshot from the deploy preview with your edits: 
![image](https://user-images.githubusercontent.com/32290/128228787-956d72ad-2a90-44e1-9ba1-95265dfe8a0a.png)

I agree that we want to CALL it extreme risk instead of dangerously high.  That makes total sense to me. But I think we want to "demote" extreme from "all red"-mode to just the normal "top of the spectrum.

Reply:
I'm going to remove this from this PR.

Reply:
This change was apparently removed in one of the force pushes.